### DISCUSSION TITLE
Github actions for MacOS, Win, Linux releases

### DISCUSSION THREAD

**Discussion by krupitskas** [dfranx/SHADERed#202] [Ideas]
Hello! I've recently set up such workflow in my repo, I'm curious, do you want to have the same? I mean on each tag push, all 3 builds will be triggered and artifacts would be uploaded.
If you are, I'm willing to help :)

**Comment by dfranx** [comment]
Hello! Thank you so much for offering the help but I don't think this is currently necessary because I don't want to provide "official" MacOS binaries myself yet, since there are several (big) features that are not implemented on MacOS + I can't test it.

**Reply by RektTillNoon** [reply]
Hey you need to update this for macos, seriously...
